<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MAP-E Address Caculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="I_v6p">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>70</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="I_v6pl">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>100</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="I_v4p">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>130</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="I_v4pl">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>160</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="I_eabitl">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>190</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="I_psidoffset">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>220</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_v6p">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>75</y>
      <width>61</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>ipv6Prefix</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_v6pl">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>105</y>
      <width>96</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>ipv6PrefixLength</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_v4p">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>135</y>
      <width>61</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>ipv4Prefix</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_v4pl">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>165</y>
      <width>96</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>ipv4PrefixLength</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_psidoffset">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>225</y>
      <width>61</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>psIdOffset</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_eabitl">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>195</y>
      <width>66</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>eaBitLength</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>305</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Caculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_eup">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>255</y>
      <width>81</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>endUserPrefix</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="I_eup">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>250</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_bin">
    <property name="geometry">
     <rect>
      <x>745</x>
      <y>35</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>BIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_v6p_b">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>716</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="L_v4p_b">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>716</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="L_eup_b">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>716</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="L_v6a">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>425</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IPv6 Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_v6a_b">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>BIN:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>345</y>
      <width>1121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>65</y>
      <width>16</width>
      <height>236</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="L_v4a">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>590</y>
      <width>81</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>IPv4 Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_v6a_v">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>425</y>
      <width>896</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="L_v6a_b_v">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>450</y>
      <width>896</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="L_v4a_b_v">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>610</y>
      <width>436</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="L_v4a_h_v">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>640</y>
      <width>436</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="L_psid">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>591</y>
      <width>36</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>PSID</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_bin_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_hex">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_dec">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>670</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DEC</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_psid_b">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>610</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="L_psid_h">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>640</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="L_psid_d">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>670</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="I_eupl">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>280</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_eupl">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>285</y>
      <width>116</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>endUserPrefixLength</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_v4a_d_v">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>670</y>
      <width>436</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
